pom.xml

```diff
@@ -23,7 +23,7 @@
     <parent>
         <groupId>org.apache.sling</groupId>
         <artifactId>sling</artifactId>
-        <version>26</version>
+        <version>39</version>
```

Comment:
Please use 'sling-bundle-parent' as artifact id (https://github.com/apache/sling-parent/blob/133c73392e9575cb10ced19ce29e14e7ec61c790/sling-bundle-parent/pom.xml#L29)

Reply:
Using the sling-bundle-parent would require moving to the bnd-maven-plugin, which I agree can be done, but I would say is not required here. It can be done as part of a different JIRA issue as discussed with @rombert above.

Reply:
Then please stick with a version <= 34. Referencing the newest generic (non-bundle) parent feels just wrong